### PR TITLE
Create et.json

### PR DESCRIPTION
**Pull request by Dh0th**
Hi! Here is a new language for electricityMap. It's Estonian.

### CODE REVIEW

**Comment by madsnedergaard** [comment]
Hi @Dh0th, this is awesome! 💪 
The new language needs to be added in a few more places to work:

- https://github.com/tmrowco/electricitymap-contrib/blob/9e25468f81e46a21293500512f66b51c02ca4358/mobileapp/www/index.html#L5
- https://github.com/tmrowco/electricitymap-contrib/blob/master/web/locales-config.json

Would you mind updating this PR with those changes? :)

**Comment by Dh0th** [comment]
@madsnedergaard Done!

File: mobileapp/www/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <script>
       document.addEventListener('deviceready', onDeviceReady, false);
       // Warning: those locales are duplicated from locales-config.json
-      var locales = ['ar', 'cs', 'da', 'de', 'en', 'es', 'fr', 'hr', ,'id', 'it', 'ja', 'nl', 'pl', 'pt-br', 'ru', 'sk', 'sv', 'vi', 'zh-cn', 'zh-hk', 'zh-tw'];
+      var locales = ['ar', 'cs', 'da', 'de', 'en', 'es', 'et', 'fr', 'hr', ,'id', 'it', 'ja', 'nl', 'pl', 'pt-br', 'ru', 'sk', 'sv', 'vi', 'zh-cn', 'zh-hk', 'zh-tw'];
```
Comment (veqtrus) [inline mobileapp/www/index.html:5]: Hmm, `el` (Greek) is missing, could you add it?

Comment (veqtrus) [reply mobileapp/www/index.html:5]: Also what's that `'hr', ,'id',`?

Comment (Dh0th) [reply mobileapp/www/index.html]: Fair enough!

**Comment by veqtrus** [comment]
This reminds me that we should implement #2737 at some point.

**Comment by madsnedergaard** [comment]
Nice catch with the double commas, makes me wonder if this thing is even used or what?! 😂

**Review by madsnedergaard — APPROVED** [review]
Looks good to me now, thanks both of you! 💚

File: mobileapp/www/index.html (hunk 1 above)
Comment (madsnedergaard) [reply mobileapp/www/index.html]: Nice catch with the double commas @veqtrus, makes me wonder if this thing is even used or if it's just been broken for a while?! 😂

**Pull request merged**